packages/host/app/components/operator-mode/container.gts

```diff
@@ -92,6 +95,9 @@ export default class OperatorModeContainer extends Component<Signature> {
   @tracked searchSheetTrigger: SearchSheetTrigger | null = null;
   @tracked isChatVisible = false;
 
+  @tracked cardForCardMode: CardDef | null = null;
+  @tracked cardNotFoundError: string | null = null;
```

Comment:
i would rename this to codeModeCardError that makes it more clear what this error is about.